util/rootless/specconv/specconv_linux.go

```diff
@@ -33,6 +38,13 @@ func ToRootless(spec *specs.Spec) error {
 	}
 	spec.Mounts = mounts
 
+	spec.Mounts = append(spec.Mounts, specs.Mount{
+		Destination: "/sys",
+		Type:        "none",
+		Source:      "/sys",
+		Options:     []string{"rbind", "nosuid", "noexec", "nodev", "ro"},
```

Comment:
This is insecure 


https://github.com/moby/buildkit/pull/1627#discussion_r498666422

Reply:
Hey, sorry, I didnt mean for you to review this yet :)  Yeah, I was trying different approaches.  This is the only one that works, and as you noted it is insecure.   I was not able to finding a way that works while also being secure, so I just tried to fix it on the `runc` side, opened a PR there: https://github.com/opencontainers/runc/pull/2634

I will likely close this PR and open another one to pull in the runc changes when can get a fix for that merged in.